app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb

```diff
@@ -1,10 +1,13 @@
 class ManageIQ::Providers::Lenovo::PhysicalInfraManager::EventCatcher::Stream
+  TIMEOUT = 90
```

Comment:
A quick comment would be welcome here, explaining the number and its quantity (seconds, minutes, hours)